src/libraries/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs

```diff
@@ -1634,6 +1634,14 @@ public partial class DesignerVerbCollection : System.Collections.CollectionBase
         protected override void OnValidate(object value) { }
         public void Remove(System.ComponentModel.Design.DesignerVerb value) { }
     }
+//    public class RuntimeLicenseContext : System.ComponentModel.LicenseContext
```

Comment:
why are we adding these comments to the ref assembly?

Reply:
Don't review yet :) I'll ping folks once it's ready !!

Reply:
Ah, instead of closing/re-opening you could downgrade it to a draft PR 😄